corehq/ex-submodules/casexml/apps/stock/consumption.py

```diff
@@ -89,8 +91,11 @@ def compute_daily_consumption(case_id,
         window_start,
         window_end
     )
+    sql_product = SQLProduct.objects.get(product_id=product_id)
+    domain = sql_product.domain
```

Comment:
actually, rather than using a toggle for this I wonder if we can add it to the domain consumption settings object?